docs/sources/sharing/share-panel.md

```diff
@@ -14,6 +14,8 @@ To share a panel:
 1. Click a panel title to open the panel menu.
 1. Click **Share**. The share dialog opens and shows the Link tab.
 
+{{< docs-imagebox img="/img/docs/sharing/Share_Panel_Direct_Link.png" max-width="700px" >}}
```

Comment:
See earlier comments about image link and file name.

Reply:
Will fix in separate PR.